packages/server/src/utils/types.ts

```diff
@@ -38,12 +38,25 @@ export interface DatabaseConfig {
 	client: DatabaseConfigClient;
 	// For Postgres, this is the actual database name. For SQLite, this is the
 	// path to the SQLite file.
-	name: string;
+	name?: string;
+	connectionString?: string;
 	host?: string;
 	port?: number;
 	user?: string;
 	password?: string;
 	asyncStackTraces?: boolean;
+	sslCaFilePath?: string;
+	sslCertFilePath?: string;
+	sslCertKeyFilePath?: string;
+	rejectUnauthorized?: boolean;
+}
+
+export interface DatabaseConfigSsl
```

Comment:
Should be DbConfigConnectionSsl

Reply:
The type above is called `DatabaseConfig` , and this is where this is used hence the name of this type. Happy to update but wanted to point out the discrepancy, should `DatabaseConfig` be updated to `DbConfig` also?

Reply:
Right, I missed that this object is used in two different places. So it's good then.